File: practices/financecalc.py
# MM 6th Financial Caluclator 

income = float(input("What is your monthly income? \n"))
rent = float(input("What is the cost of your rent/mortgage? \n"))
utilities = float(input("How much do you typically spend on utilities? \n"))
groceries = float(input("How much do you typically spend on groceries? \n"))
transport = float(input("How much do you spend on transportation? \n"))
left = round(income - rent - utilities - groceries - transport, 2)

print("Your rent/mortgage is $", rent, ", which is", round( round((rent / income), 3) * 100, 2), "% of your total income.")
print("Your utilities are $", utilities, ", which is", round( round((utilities / income), 3) * 100, 2), "% of your total income.")
print("Your groceries are $", groceries, ", which is", round( round((groceries / income), 3) * 100, 2), "% of your total income.")
print("Your transportation costs $", transport, ", which is", round( round((transport / income), 3) * 100, 2), "% of your total income.")
print("You have $", left, " to spend freely each month!")

File: c_programming/practices/finance_calc.c
// MM 6th CSP Financial Calculator
#include <stdio.h>

int main(void){

    float monthly_income;
    float rent;
    float utilities;
    float groceries;
    float transport;
    printf("What is your monthly income?\n");
    scanf("%f", &monthly_income);
    printf("What is your rent/mortgage?\n");
    scanf("%f", &rent);
    printf("How much do you typically spend on utilities?\n");
    scanf("%f", &utilities);
    printf("How much do you typically spend on groceries?\n");
    scanf("%f", &groceries);
    printf("How much do you typically spend on transport?\n");
    scanf("%f", &transport);

    float left = monthly_income - rent - utilities - groceries - transport;

    printf("Your rent/mortgage is %1.2f, which is %1.2f%% of your income.", 4.11, 4.11);
    return 0;
}
